app/views/steps/interstitial/show.html.erb

```diff
@@ -3,7 +3,11 @@
 <span class="govuk-caption-xl">Calculate import duties</span>
 
 <% if user_session.gb_to_ni_route? %>
-  <%= render partial: 'steps/interstitial/shared/context', locals: { heading: t('interstitial.gb_to_ni.heading'), body: t('interstitial.gb_to_ni.body') } %>
+  <% if user_session.trade_defence? %>
```

Comment:
I'm going to simplify this with an interstitial helper: https://transformuk.atlassian.net/browse/HOTT-1083.

A view shouldn't have this much logic in it IMO